commcare_connect/templates/opportunity/opportunity_detail.html

```diff
@@ -114,27 +114,64 @@ <h1 class="display-5 mb-0">{{ object.name }}</h1>
         </tr>
         </tbody>
       </table>
-      <h4>Learn App Modules</h4>
-
-      <table class="table border table-custom">
-        <thead class="table-light">
-        <tr>
-          <th>Name</th>
-          <th>Description</th>
-          <th>Time Estimate</th>
-        </tr>
-        </thead>
-        <tbody>
-        {% for module in object.learn_app.learn_modules.all %}
-          <tr>
-            <td>{{ module.name }}</td>
-            <td>{{ module.description }}</td>
-            <td>{{ module.time_estimate }} hour{{ module.time_estimate|pluralize:",s" }}</td>
-          </tr>
-        {% endfor %}
-        </tbody>
-      </table>
-      <hr/>
+      <div class="accordion" id="accordionExample">
+        <div class="accordion-item">
+          <h2 class="accordion-header" id="headingOne">
+            <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#learn-app-modules" aria-expanded="true" aria-controls="collapseOne">
+              Learn App Modules
+            </button>
+          </h2>
+          <div id="learn-app-modules" class="accordion-collapse collapse show" aria-labelledby="headingOne" data-bs-parent="#accordionExample">
+            <div class="accordion-body p-0 m-0">
+              <table class="table table-borderless table-custom">
+                <thead class="table-light">
+                <tr>
+                  <th>Name</th>
+                  <th>Description</th>
+                  <th>Time Estimate</th>
+                </tr>
+                </thead>
+                <tbody>
+                {% for module in object.learn_app.learn_modules.all %}
+                  <tr>
+                    <td>{{ module.name }}</td>
+                    <td>{{ module.description }}</td>
+                    <td>{{ module.time_estimate }} hour{{ module.time_estimate|pluralize:",s" }}</td>
+                  </tr>
+                {% endfor %}
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </div>
+        <div class="accordion-item">
+          <h2 class="accordion-header" id="headingTwo">
+            <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#deliver-units" aria-expanded="false" aria-controls="collapseTwo">
+              Deliver Units
+            </button>
+          </h2>
+          <div id="deliver-units" class="accordion-collapse collapse" aria-labelledby="headingTwo" data-bs-parent="#accordionExample">
+            <div class="accordion-body p-0 m-0">
+              <table class="table table-borderless table-custom">
+                <thead class="table-light">
+                <tr>
+                  <th>Name</th>
+                  <th>Slug</th>
```

Comment:
Nit: I think this column should be called "ID". That is a much more understandable name to non developers, and matches what the field is called in the form builder.

Reply:
Renamed. https://github.com/dimagi/commcare-connect/pull/157/commits/6fc12416796f2f55a9fd97c43da8fc1a0f55ae5b

This column shows the ID for the deliver units. I have attached a screenshot with the correct table.